README.rst

```diff
@@ -626,7 +626,7 @@ may be re-created automatically if the system needs it.
 Silencing notifications for specific objects temporarily or permanently
 -----------------------------------------------------------------------
 
-.. image:: https://github.com/openwisp/openwisp-notifications/raw/docs/docs/images/silence-notifications.png
+.. image:: https://user-images.githubusercontent.com/56113566/163180707-78abfb33-22d3-4265-950e-4c47fa348891.png
```

Comment:
what's going on here?

Reply:
I'm trying to change [this images](https://github.com/openwisp/openwisp-notifications/raw/docs/docs/images/silence-notifications.png) with [new one](https://user-images.githubusercontent.com/56113566/163180707-78abfb33-22d3-4265-950e-4c47fa348891.png) using github-user-content.
Just found that we have a separate [docs branch](https://github.com/openwisp/openwisp-notifications/tree/docs) for managing docs image. So it's better to changes there.

Reply:
@Aryamanz29 I have uploaded your image in docs branch: https://github.com/openwisp/openwisp-notifications/commit/c829a4316b952a0460e9436c74e3404c6738581c

You can change the url now. :+1: